src/sample/classes/Student.java

```diff
@@ -16,7 +16,7 @@ public static void main(String[] args) throws FileNotFoundException{
         list1.add("Kotlin");
         list1.add("Android");
         
-        String path = "Desktop/CourseList.pdf"; 
+        String path = "C:/Users/User/Downloads/CourseList.pdf"; 
```

Comment:
I don't think everyone has `users/user` path, is it possible to allow users to choose where to download?

Reply:
i am not sure about this, if let user choose where to download means will involve php dy isnt it? maybe we can discuss a way forward after 232 class

Reply:
Are you going to change this @YouQuan2007 ?

Reply:
@ziqiang2344 have solve this issue dy, so i guess this pull request can delete